core/src/services/s3/core.rs

```diff
@@ -394,42 +379,6 @@ impl S3Core {
         // Set SSE headers.
         req = self.insert_sse_headers(req, true);
 
-        if let Some(v) = &self.server_side_encryption_customer_algorithm {
```

Comment:
It's already included in `self.insert_sse_headers`, so removed here.

Reply:
`X_AMZ_COPY_SOURCE_SERVER_SIDE_ENCRYPTION_CUSTOMER_ALGORITHM` is different other SSE headers. They should not be removed.